packages/@react-aria/interactions/src/useInteractOutside.ts

```diff
@@ -123,5 +123,14 @@ function isValidEvent(event, ref) {
     }
   }
 
-  return ref.current && !ref.current.contains(event.target);
+  if (!ref.current) {
+    return false;
+  }
+
+  const refWithChildren = [ref.current, ...Array.from(ref.current.childNodes)];
```

Comment:
apologies for my lack of knowledge here, I'm sure I'm just overlooking something, but why do we need the childNodes?

Reply:
This is actually a good question. I was just using the old `ref.current.contains` (which checks childNodes as well). However, since we're now using `target.composedPath`, if a child of the `ref.current` was clicked, that _should_ be present in the composedPath. I think we can actually just use `ref.current`